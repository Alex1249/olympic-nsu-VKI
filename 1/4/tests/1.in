10 8
0
1
2
4
7
5
6
9
