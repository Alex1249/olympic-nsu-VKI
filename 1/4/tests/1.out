0 not
1 not
2 prime
4 not
7 prime
5 prime
6 not
9 not
